test/index_test.js

```diff
@@ -56,23 +72,40 @@ describe("App", () => {
 
   describe("Store", () => {
 
+    beforeEach((done) => {
+      sandbox.stub(store.collection, 'create')
+        .returns(Promise.resolve({data: {label: "Hola!"}}));
```

Comment:
You should add dedicated `describe` blocks for each public methods, and move specific stubbing down there.